catroid/config/detekt.yml

```diff
@@ -1,29 +1,5 @@
 autoCorrect: true
 
-test-pattern: # Configure exclusions for test sources
```

Comment:
Test pattern has been migrated according to this guideline:
https://arturbosch.github.io/detekt/howto-migratetestpattern.html